src/osd/PGLog.h

```diff
@@ -476,18 +476,14 @@ struct PGLog : DoutPrefixProvider {
     }
 
     void index(pg_log_dup_t& e) {
-      if (PGLOG_INDEXED_DUPS) {
```

Comment:
Kefu's fix is correct. It should be:

`if (indexed_data & PGLOG_INDEXED_DUPS) {`